### DISCUSSION TITLE
Create portfolio by using this a template

### DISCUSSION THREAD

**Discussion by DFpara** [BRAVO68WEB/bravo68web-portfolio-nextjs#8] [Q&A]
How can I take on you project to create my very own portfolio?

**Comment by BRAVO68WEB** [comment]
## Dev Guidance (Step-by-Step)

- Clone the repository. 
- Install all dependencies
- Do a test run via ```yarn dev```
- Check ``config.js`` and do change accordingly
- disable the features/components not needed by you

_The API used here not public but ask me to help you out for the same if required._

**Reply by DFpara** [reply]
Sure thx
